### PR TITLE
ADST-598

### PR DESCRIPTION
**Pull request by aman-alfresco**
updated dependencies to newer versions. code fixing.

### CODE REVIEW

File: sample/src/main/java/com/alfresco/sample/MainViewModel.kt [hunk 1]
```diff
@@ -112,254 +94,20 @@ class MainViewModel(private val context: Context) : ViewModel() {
...
-                val taskList = serviceAPS1.createProcessInstance(
-                    RequestProcessInstances(
-                        name = "Test process aman",
-                        processDefinitionId = "Testprocessaman:2:51",
-                        values = mapOf(Pair("firstname", "Amanpal"), Pair("lastname", "Singh"))
-
-                    )
                 )
 
+                println("Response recent data = $searchCall")
```
Comment (PiplaniVinay) [inline sample/src/main/java/com/alfresco/sample/MainViewModel.kt:109]: Please remove the  println.

File: auth/build.gradle [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+    id 'kotlin-android'
+    id 'kotlinx-serialization'
+    id 'org.jetbrains.dokka'
+}
 
 android {
     namespace 'com.alfresco.auth'
     defaultConfig {
-        versionName "0.8.1"
+        versionName "0.8.3"
```
Comment (PiplaniVinay) [inline auth/build.gradle:11]: The version should be 0.8.2.

Comment (aman-alfresco) [reply auth/build.gradle:11]: 0.8.2 already released for auth0 changes

**Review by PiplaniVinay — APPROVED** [review]

File: sample/src/main/java/com/alfresco/sample/AuthConfig.kt [hunk 3]
```diff
@@ -6,8 +6,8 @@ val AuthConfig.Companion.defaultConfig: AuthConfig
     get() = AuthConfig(
         https = true,
         port = "443",
-        clientId = "zTSfC3VEcrguvP5CKIgNIBCOTliawIKg",
-        realm = "",
-        redirectUrl = "demo://dev-ps-alfresco.auth0.com/android/com.alfresco.sample/callback",
+        clientId = "alfresco-android-acs-app",
```
Comment (gl-lovekesh) [inline sample/src/main/java/com/alfresco/sample/AuthConfig.kt:9]: These values should replace with placeholder, it should not contain demo server specific value

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

**Review by PiplaniVinay — APPROVED** [review]

**Pull request merged**